src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -405,10 +404,6 @@ public void hoverClickAndViewPhotoOnTableCell(int questionBodyIndex, int tableRo
         
         waitForElementPresence(By.cssSelector(".popover-content"));
         
-        WebElement popoverContent = browser.driver.findElement(By.className("popover-content"));
-        
-        waitForElementVisibility(popoverContent);
-        
```

Comment:
These are added in #4702 which is a failed attempt to remove the timeout exception, reason being what I wrote below: `document.getElementsByClassName` (or by any other selector) works simply with presence and not visibility, so these lines contribute nothing.
"Visibility" is treated differently in different browsers, some treat it as "present and not hidden", some add a condition that is "can be seen in the viewport", but I'm in no position to know everything about which browser is which. Quite apparently the one used by Travis is the latter.